client/my-sites/email/titan-setup-thank-you/index.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ThankYou } from 'calypso/components/thank-you';
+import theme from 'calypso/my-sites/marketplace/theme';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getTitanEmailUrl } from 'calypso/lib/titan';
+import Gridicon from 'calypso/components/gridicon';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+
+type TitanSetupThankYouProps = {
+	domainName: string;
+	emailAddress?: string;
+};
+
+export const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
+	const currentRoute = useSelector( getCurrentRoute );
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const translate = useTranslate();
+	const thankYouImageSrc = '/calypso/images/upgrades/thank-you.svg';
+
+	const { domainName, emailAddress = `youremail@${ domainName }` } = props;
+
+	const emailManagementPath = emailManagement( selectedSiteSlug, domainName, currentRoute );
+
+	const thankYouImage = {
+		alt: 'Thank you',
+		src: thankYouImageSrc,
+	};
+
+	const titanThankYouSection = {
+		sectionKey: 'titan_whats_next',
+		sectionTitle: translate( 'What’s next?' ),
+		nextSteps: [
+			{
+				stepKey: 'titan_whats_next_view_inbox',
+				stepTitle: translate( 'Access your inbox' ),
+				stepDescription: translate( 'Access your email from anywhere with our built-in webmail.' ),
+				stepCta: (
+					<FullWidthButton href={ getTitanEmailUrl( emailAddress ) } primary>
+						{ translate( 'Go to Inbox' ) }
+						<Gridicon icon="external" />
+					</FullWidthButton>
+				),
+			},
+			{
+				stepKey: 'titan_whats_next_get_mobile_app',
+				stepTitle: translate( 'Get mobile app' ),
+				stepDescription: translate(
+					"Access your email on the go with Titan's Android and iOS apps."
```

Comment:
I was about to mention that we should refer to `Professional Email` instead of `Titan` but after giving it more thoughts this is probably one of the few places where `Titan` is a better option - users will download a `Titan` app after all. However if we keep it here, we should probably add it on the `Manage All Mailboxes` page then:

<img width="622" alt="screenshot" src="https://user-images.githubusercontent.com/594356/127189067-0f47d47b-6634-4d35-81c9-da457a25dbcb.png">

Reply:
The string in the `Manage All Mailboxes` page has not been updated, it should be:

> Download Titan's Android and iOS apps to access your emails on the go